libsolidity/interface/FileReader.cpp

```diff
@@ -31,9 +33,17 @@ using std::string;
 namespace solidity::frontend
 {
 
+void FileReader::setBasePath(boost::filesystem::path const& _path)
+{
+	m_basePath = (_path.empty() ? "" : normalizeCLIPathForVFS(_path));
```

Comment:
What about setting `normalizeCLIPathForVFS(".")` already here, if `_path` is empty? With this `m_basePath` would always contain a valid path, that is also always the base path.

Reply:
The problem is that `""` is a special case that is not equivalent to any specific path.

For example if I use the working dir as you're suggesting, it will break absolute paths - `import "/x/y.sol"` will try to load `x.y.sol` from work dir rather than the absolute `/x/y.sol`.

Using `/` would keep absolute paths working but it would relative paths - `import "x/y.sol"` would try to load `/x.y.sol`.

So these two values are out and I can't think of any other path that would keep `--base-path=""` working as it does now.